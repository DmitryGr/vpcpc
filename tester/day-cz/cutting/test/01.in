39
75 473
91 472
58 468
114 461
123 451
33 440
132 430
30 419
129 408
124 397
47 387
106 380
95 376
82 374
78 358
82 345
65 338
57 331
78 325
78 318
118 312
78 305
132 299
139 292
78 283
78 262
78 241
78 220
74 204
86 194
92 183
55 172
105 161
111 150
36 139
123 129
23 118
135 108
11 97
