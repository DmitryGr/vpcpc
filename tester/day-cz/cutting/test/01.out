0 318 -2 318
82 0 82 2
-146 146 -147 145
-1 183 -3 183
123 1 123 3
-84 83 -85 82
0 440 -2 440
36 0 36 2
75 1 75 3
-137 137 -138 136
106 0 106 2
-54 54 -55 53
0 118 -2 118
159 160 158 161
-1 283 -3 283
-124 123 -125 122
191 192 190 193
